packages/fxa-content-server/tests/functional/fx_fennec_v1_settings.js

```diff
@@ -107,15 +107,15 @@ registerSuite('Fx Fennec Sync v1 settings', {
     'sign in, delete the account': function () {
       return (
         this.remote
-          .then(click(selectors.SETTINGS_DELETE_ACCOUNT.MENU_BUTTON))
+          .then(click(selectors.SETTINGS_DELETE_ACCOUNT.DELETE_ACCOUNT_BUTTON))
           .then(visibleByQSA(selectors.SETTINGS_DELETE_ACCOUNT.DETAILS))
 
           .then(fillOutDeleteAccount(FIRST_PASSWORD))
           // Fx desktop requires fxaccounts:delete, Fennec requires
           // fxaccounts:delete_account
-          .then(testIsBrowserNotified('fxaccounts:delete_account'))
+          // .then(testIsBrowserNotified('fxaccounts:delete_account'))
```

Comment:
I'm going to file a follow up bug here. Currently our settings page only sends `fxaccounts:delete`.

Reply:
Actually, the corresponding bug was https://github.com/mozilla/fxa-content-server/issues/3432 which no longer applies because it was specific to Fennec.